_posts/2021-09-10-anna-makarudze.md

```diff
@@ -0,0 +1,18 @@
+---
+author: Lacey Williams Henschel and Drew Wisntel
+category: General
+date: 2021-09-10 15:00:00
+layout: post
+image: /static/img/speakers/anna-makarudze.jpg
+title: "Announcing Our Invited Speaker: Anna Makarudze"
+---
+
+We are so excited to announce our invited speaker for DjangoCon US 2021: Anna Makarudze!
+
+<img src="/static/img/speakers/anna-makarudze.jpg" alt="Photo of Anna Makarudze" style="width:45%; display:block; float:right;" />
+
+Anna is currently the President of the [Django Software Foundation](https://www.djangoproject.com/foundation/) Board of Directors for 2021. She was previously Fundraising Coordinator for the [Django Girls Foundation](https://djangogirls.org/) and has given talks at [DjangoCon US 2018](https://2018.djangocon.us/talk/keynote-with-anna-makarudze/), DjangoCon Europe, PyCon Italia, PyCon Namibia, PyCon South Africa, and many others. She has also been on the organizing team for many DjangoGirls events in Zimbabwe, the first two PyCon Zimbabwe events, and DjangoCon Europe 2018. She also runs [PyLadies Harare](https://twitter.com/PyladiesHRE) and is a founder trustee of the Zimbabwe Python Association.
```

Comment:
```suggestion
Anna is the President of the [Django Software Foundation](https://www.djangoproject.com/foundation/) Board of Directors and is the Fundraising Coordinator for the [Django Girls Foundation](https://djangogirls.org/) and has given talks at [DjangoCon US 2018](https://2018.djangocon.us/talk/keynote-with-anna-makarudze/), DjangoCon Europe, PyCon Italia, PyCon Namibia, PyCon South Africa, and many others. She has also been on the organizing team for many DjangoGirls events in Zimbabwe, the first two PyCon Zimbabwe events, and DjangoCon Europe 2018. She also ran [PyLadies Harare](https://twitter.com/PyladiesHRE) and is a founder trustee of the Zimbabwe Python Association.
```

Reply:
@drewbrew I added a suggestion based on @amakarudze's feedback but feel free to accept, re-word, etc.